includes/Modules/AdSense.php

```diff
@@ -197,6 +202,7 @@ public function get_debug_fields() {
 	protected function get_datapoint_definitions() {
 		return array(
 			'GET:accounts'       => array( 'service' => 'adsense' ),
+			'GET:adunits'        => array( 'service' => 'adsense' ),
```

Comment:
@benbowler could you please move this to be after `GET:alerts` since we keep all datapoint definitions sorted alphabetically?  Same for the new handler added below.